docs/Reference.md

```diff
@@ -1504,6 +1505,19 @@ An object that specifies the credentials of a user to be used for basic authenti
 | ------ | -------- |
 | object | No       |
 
+### `useWebView2`
+
+Use WinUI WebView2 control instead of WebView control as the native webview. The WebView2 control is a WinUI control that renders web content using the Microsoft Edge (Chromium) rendering engine.
+
+| Type    | Required | Platform |
+| ------- | -------- | -------- |
+| boolean | No       | Windows  |
+
+Example:
+
+```javascript
+<WebView useWebView2={true} />
```

Comment:
is it expected that this value can be toggled at runtime? e.g.

```jsx
const [wv2, setwv2] = useState(false);
// ...
/// something happens and calls setwv2(true)
// ...
<WebView useWebView2={wv2} />
```

Reply:
Yes toggle at runtime and works with fast refresh! I'll add to the option description to make that clearer.